modules/light-clients/08-wasm/types/vm.go

```diff
@@ -26,16 +26,17 @@ var (
 	// wasmvmAPI is a wasmvm.GoAPI implementation that is passed to the wasmvm, it
 	// doesn't implement any functionality, directly returning an error.
 	wasmvmAPI = wasmvm.GoAPI{
-		HumanAddress:     humanAddress,
-		CanonicalAddress: canonicalAddress,
+		HumanizeAddress:     humanizeAddress,
+		CanonicalizeAddress: canonicalizeAddress,
+		ValidateAddress:     validateAddress,
```

Comment:
Is this called by something outside our reach? I see it returns an error, same with the other two above. I guess this is intended not to be called by contracts?

Reply:
To avoid null pointer crashes further down the road, latest versions of wasmvm requires all the fields to be set to a non-nil value